src/micro_http/src/common/headers.rs

```diff
@@ -416,6 +438,17 @@ mod tests {
         assert!(header
             .parse_header_line(b"Content-Type: application/json")
             .is_ok());
+
+        // Test valid accept media type.
+        assert!(header
```

Comment:
We should check the value of the header instead of just `is_ok`.

Reply:
Done.